src/cljs/main/broadfcui/page/workspace/monitor/submission_details.cljs

```diff
@@ -186,7 +186,20 @@
              [:span {:style {:fontWeight 500}} (:methodConfigurationNamespace submission)]]
             [:div {}
              [:div {:style {:fontWeight 200 :display "inline-block" :width 90}} "Name:"]
-             [:span {:style {:fontWeight 500}} (:methodConfigurationName submission)]])
+             ;; if we were able to retrieve the method config (see :load-details), then display
+             ;; the config's name as a link. Else, display it as text with an informative tooltip.
+             (if (:ws-config @state)
```

Comment:
Destructure state and props

Reply:
Also wrap this in a let and destructure the things you're using from `submission`